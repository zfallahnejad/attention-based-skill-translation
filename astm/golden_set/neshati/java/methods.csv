22656,methods
1707091,methods
992484,methods
335858,methods
522444,methods
1679863,methods
438154,methods
535871,methods
1065197,methods
276052,methods
1103872,methods
157247,methods
829571,methods
203907,methods
