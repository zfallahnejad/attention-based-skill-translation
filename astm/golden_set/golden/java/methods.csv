22656,54,54,0.5
1707091,34,47,0.41975308641975306
335858,26,28,0.48148148148148145
1679863,26,20,0.5652173913043478
992484,26,30,0.4642857142857143
522444,26,36,0.41935483870967744
438154,20,22,0.47619047619047616
535871,18,23,0.43902439024390244
1065197,16,14,0.5333333333333333
276052,16,17,0.48484848484848486
157247,14,5,0.7368421052631579
1103872,14,14,0.5
829571,13,9,0.5909090909090909
203907,11,16,0.4074074074074074
